uint8 STOPPED = 1
uint8 NODE_CONFIGURED = 2
uint8 COMPONENT_CONNECTED = 3
uint8 COMPONENT_RUNNING = 4
uint8 COMPONENT_PAUSED = 5
uint8 COMPONENT_DISCONNECTED = 6